<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>818</width>
     <height>557</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>7</number>
      </property>
      <item row="7" column="7">
       <widget class="QPushButton" name="DrawTube_Button">
        <property name="text">
         <string>Draw Tube</string>
        </property>
       </widget>
      </item>
      <item row="8" column="7">
       <widget class="QPushButton" name="DrawDoughnut_Button">
        <property name="text">
         <string>Draw Doughnut</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="TranslateCube_Button">
        <property name="text">
         <string>Translate Cube</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="FlipHemisphere_Button">
        <property name="text">
         <string>Flip HemiSphere</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="TranslateHemisphere_Button">
        <property name="text">
         <string>Translate Hemisphere</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="ScalePyramid_Button">
        <property name="text">
         <string>Scale Pyramid</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QPushButton" name="ChangePyramidOpacity_Button">
        <property name="text">
         <string>Change Pyramid Opacity</string>
        </property>
       </widget>
      </item>
      <item row="6" column="6">
       <widget class="QPushButton" name="RemoveCylinder_Button">
        <property name="text">
         <string>Remove Cylinder</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="ChangeCubeOpacity_Button">
        <property name="text">
         <string>Change Cube Opacity</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="ScaleCone_Button">
        <property name="text">
         <string>Scale Cone</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="RotatePyramid_Button">
        <property name="text">
         <string>Rotate Pyramid</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="RotateCone_Button">
        <property name="text">
         <string>Rotate Cone</string>
        </property>
       </widget>
      </item>
      <item row="6" column="7">
       <widget class="QPushButton" name="DrawCylinder_Button">
        <property name="text">
         <string>Draw Cylinder</string>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QPushButton" name="DrawCone_Button">
        <property name="text">
         <string>Draw Cone</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="ChangeCubeColor_Button">
        <property name="text">
         <string>Change Cube Color</string>
        </property>
       </widget>
      </item>
      <item row="9" column="7">
       <widget class="QPushButton" name="DrawCurvedCylinder_Button">
        <property name="text">
         <string>Draw Curved Cylinder</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="TranslateCone_Button">
        <property name="text">
         <string>Translate Cone</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QPushButton" name="RemoveHemiSphere_Button">
        <property name="text">
         <string>Remove HemiSphere</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="RotateHemisphere_Button">
        <property name="text">
         <string>Rotate HemiSphere</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="TranslateSphere_Button">
        <property name="text">
         <string>Translate Sphere</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QPushButton" name="ChangeHemisphereColor_Button">
        <property name="text">
         <string>Change HemiSphere Color</string>
        </property>
       </widget>
      </item>
      <item row="7" column="6">
       <widget class="QPushButton" name="RemoveTube_Button">
        <property name="text">
         <string>Remove Tube</string>
        </property>
       </widget>
      </item>
      <item row="5" column="7">
       <widget class="QPushButton" name="DrawPyramid_Button">
        <property name="text">
         <string>Draw Pyramid</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="ScaleCube_Button">
        <property name="text">
         <string>Scale Cube</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="ScaleSphere_Button">
        <property name="text">
         <string>Scale Sphere</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="TranslatePyramid_Button">
        <property name="text">
         <string>Translate Pyramid</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="TranslateCylinder_Button">
        <property name="text">
         <string>Translate Cylinder</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="ChangeSphereColor_Button">
        <property name="text">
         <string>Change Sphere color</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="ScaleHemisphere_Button">
        <property name="text">
         <string>Scale Hemisphere</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QPushButton" name="ChangeCylinderOpacity_Button">
        <property name="text">
         <string>Change Cylinder Opacity</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="RotateCylinder_Button">
        <property name="text">
         <string>Rotate Cylinder</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QPushButton" name="RemoveCone_Button">
        <property name="text">
         <string>Remove Cone</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="ChangeSphereOpacity_Button">
        <property name="text">
         <string>Change Sphere Opacity</string>
        </property>
       </widget>
      </item>
      <item row="5" column="6">
       <widget class="QPushButton" name="RemovePyramid_Button">
        <property name="text">
         <string>Remove Pyramid</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="FlipPyramid_Button">
        <property name="text">
         <string>Flip Pyramid</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="RemoveSphere_Button">
        <property name="text">
         <string>Remove Sphere</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="FlipCube_Button">
        <property name="text">
         <string>Flip Cube</string>
        </property>
       </widget>
      </item>
      <item row="8" column="6">
       <widget class="QPushButton" name="RemoveDoughnut_Button">
        <property name="text">
         <string>Remove Doughnut</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QPushButton" name="ChangePyramidColor_Button">
        <property name="text">
         <string>Change Pyramid Color</string>
        </property>
       </widget>
      </item>
      <item row="9" column="6">
       <widget class="QPushButton" name="RemoveCurvedCylinder_Button">
        <property name="text">
         <string>Remove Curved Cylinder</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="ChangeConeColor_Button">
        <property name="text">
         <string>Change Cone Color</string>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QPushButton" name="DrawHemiSphere_Buttom">
        <property name="text">
         <string>Draw HemiSphere</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QPushButton" name="DrawCube_Button">
        <property name="text">
         <string>Draw Cube</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="RotateSphere_Button">
        <property name="text">
         <string>Rotate Sphere</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="FlipCylinder_Button">
        <property name="text">
         <string>Flip Cylinder</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="ScaleCylinder_Button">
        <property name="text">
         <string>Scale Cylinder</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="FlipSphere_Button">
        <property name="text">
         <string>Flip Sphere</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="ChangeHemisphereOpacity_Button">
        <property name="text">
         <string>Change HemiSphere Opacity</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QPushButton" name="ChangeCylinderColor_Button">
        <property name="text">
         <string>Change Cylinder Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="RemoveCube_Button">
        <property name="text">
         <string>Remove Cube</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="DrawSphere_Button">
        <property name="text">
         <string>Draw Sphere</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="FlipCone_Button">
        <property name="text">
         <string>Flip Cone</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="RotateCube_Button">
        <property name="text">
         <string>Rotate Cube</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="ChangeConeOpacity_Button">
        <property name="text">
         <string>Change Cone Opacity</string>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="QPushButton" name="ChangeTubeColor_Button">
        <property name="text">
         <string>Change Tube Color</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QPushButton" name="ChangeTubeOpacity_Button">
        <property name="text">
         <string>Change Tube Opacity</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="RotateTube_Button">
        <property name="text">
         <string>Rotate Tube</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="FlipTube_Button">
        <property name="text">
         <string>Flip Tube</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="TranslateTube_Button">
        <property name="text">
         <string>Translate Tube</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="ScaleTube_Button">
        <property name="text">
         <string>Scale Tube</string>
        </property>
       </widget>
      </item>
      <item row="8" column="5">
       <widget class="QPushButton" name="ChangeDoughnutColor_Button">
        <property name="text">
         <string>Change Doughnut Color</string>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QPushButton" name="ChangeDoughnutOpacity_Button">
        <property name="text">
         <string>Change Doughnout Opacity</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QPushButton" name="RotateDoughnut_Button">
        <property name="text">
         <string>Rotate Doughnut</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="FlipDoughnut_Button">
        <property name="text">
         <string>Flip Doughnut</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="TranslateDoughnut_Button">
        <property name="text">
         <string>Translate Doughnut</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="ScaleDoughnut_Button">
        <property name="text">
         <string>Scale Doughnut</string>
        </property>
       </widget>
      </item>
      <item row="9" column="5">
       <widget class="QPushButton" name="ChangeCurvedcylinderColor_Button">
        <property name="text">
         <string>Change CurvedCylinder Color</string>
        </property>
       </widget>
      </item>
      <item row="9" column="4">
       <widget class="QPushButton" name="ChangeCurvedcylinderOpacity_Button">
        <property name="text">
         <string>Change Curvedcylinder Opacity</string>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QPushButton" name="RotateCurvedcylinder_Button">
        <property name="text">
         <string>Rotate Curvedcylinder</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QPushButton" name="FlipCurvedcylinder_Button">
        <property name="text">
         <string>Flip Curvedcylinder</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QPushButton" name="TranslateCurvedcylinder_Button">
        <property name="text">
         <string>Translate Curvedcylinder</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="ScaleCurvedcylinder_Button">
        <property name="text">
         <string>Scale Curvedcylinder</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QVTKOpenGLNativeWidget" name="openGLWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">qvtkopenglnativewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
